completions/bash/podman

```diff
@@ -2696,7 +2695,7 @@ _podman_ps() {
 	  --pod -p
 	  --quiet -q
 	  --size -s
-	  --storage
+	  --external
```

Comment:
since you've another nit to fix, can you alphabetize this please?  Also namespace on the line below.

Reply:
I would prefer to do this in a separate PR.  Lets go through the entire contrib and alphabetize it all.